quickstarts/microsoft.maintenance/maintenance-configuration-assignments/prereqs/prereq.azuredeploy.json

```
@@ -0,0 +1,121 @@
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "maintenanceConfigurationName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Maintenance Configuration"
+      }
+    }
+  },
+  "variables": {
+    "nicName": "myVMNic",
+    "vmName": "MyWindowsVM",
+    "maintenanceConfigurationId":"[concat(resourceGroup().Id, '/providers/Microsoft.Maintenance/maintenanceConfigurations/', parameters('maintenanceConfigurationName'))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2013-04-01",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[variables('vmName')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/networkInterfaces/', variables('nicName'))]"
+      ],
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "[parameters('vmSize')]"
+        },
+        "osProfile": {
+          "computerName": "[variables('vmName')]",
+          "adminUsername": "[parameters('adminUsername')]",
+          "adminPassword": "[parameters('adminPassword')]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "MicrosoftWindowsServer",
+            "offer": "WindowsServer",
+            "sku": "2019-Datacenter",
+            "version": "latest"
+          },
+          "osDisk": {
+            "createOption": "FromImage"
+          },
+          "dataDisks": [
+            {
+              "diskSizeGB": "1023",
+              "lun": 0,
+              "createOption": "Empty"
+            }
+          ]
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces',variables('nicName'))]"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Maintenance/maintenanceConfigurations",
+      "apiVersion": "2023-04-01",
+      "name": "[variables('maintenanceConfigurationName')]",
+      "location": "[parameters('location')]",
+      "tags": {
+      },
+      "properties": {
+        "extensionProperties": {},
+        "installPatches": {
+          "linuxParameters": {
+            "classificationsToInclude": [ "Critical", "Security" ] 
+          },
+          "rebootSetting": "IfRequired",
+          "windowsParameters": {
+            "classificationsToInclude": [ "Critical", "Security" ]
+          }
+        },
+        "maintenanceScope": "InGuestPatch",
+        "maintenanceWindow": {
+          "duration": "01:55",
+          "expirationDateTime": "2023-09-16 15:00",
+          "recurEvery": "1Day",
+          "startDateTime": "2023-08-16 15:00",
+          "timeZone": "India Standard Time"
+        },
+        "visibility": "Custom"
+      }
+    }
+  ],
+  "outputs": {
+    "vmName": {
+      "type": "string",
+      "value": "[variables('vmName')]"
+    },
+    "maintenanceConfigurationId":{
```

Same change in line 118 also
```suggestion
    "maintenanceConfigurationName":{
```